rxbindings/src/main/java/com/amplifyframework/rx/RxApiBinding.java

```diff
@@ -168,4 +173,83 @@
             CancelableBehaviors.StreamEmitter<String, T, ApiException> method) {
         return CancelableBehaviors.toObservable(method);
     }
+
+    /**
+     * A class that represents a subscription operation and exposes
+     * observables for consumers to listen to subscription data and
+     * status events.
+     * @param <T> The type representing the subscription data.
+     */
+    public static final class RxSubscriptionOperation<T> implements Cancelable {
```

Comment:
does it need to be public?

Reply:
This one does, but now lives under RxOperations.